communityMUD/typeclasses/chargen.py

```diff
@@ -0,0 +1,561 @@
+"""
+Character Generation System
+
+This module handles character generation after account creation.
+
+The workflow is:
+1. Choose your character's name
+2. Enter email (optional)
+3. Select race (Human for now)
+4. Select class (Warrior for now)
+5. Confirm choices or restart
+
+"""
+
+import re
+from evennia import EvMenu
+from evennia.utils import dedent
+from typeclasses.races import apply_race, get_available_races
+from typeclasses.classes import apply_class, get_available_classes
+
+
+def node_welcome(caller, raw_text, **kwargs):
+    """
+    Welcome screen for character generation
+    """
+    text = dedent("""
+    |gWelcome to Character Creation!|n
+    
+    You will now create your character by following these steps:
+    1. Choose your character's name
+    2. Enter your email (optional)
+    3. Choose your race
+    4. Choose your character class
+    5. Review and confirm your choices
+    
+    Press |wenter|n to begin.
+    """)
+    
+    options = {
+        "key": "_default",
+        "goto": "node_name"
+    }
+    
+    return text, options
+
+
+def node_name(caller, raw_text, **kwargs):
+    """
+    Character name selection screen
+    """
+    def _check_input(caller, raw_string, **kwargs):
+        if not raw_string.strip():
+            caller.msg("Please enter a name for your character.")
+            return None
+            
+        name = raw_string.strip()
+        
+        # Basic name validation
+        if len(name) < 2:
+            caller.msg("Name must be at least 2 characters long.")
+            return None
+            
+        if len(name) > 16:
+            caller.msg("Name must be less than 16 characters long.")
```

Comment:
```suggestion
        if len(name) > 16:
            caller.msg("Name must be up to 16 characters long.")
```

Based on the check here and the text underneath this seems like a typo